src/bp/sdk/botpress.d.ts

```diff
@@ -457,7 +457,7 @@ declare module 'botpress/sdk' {
   }
 
   export namespace NLU {
-    export type EntityType = 'system' | 'pattern' | 'list'
+    export type EntityType = 'system' | 'pattern' | 'list' | 'complex'
```

Comment:
I don't thing we should define a entity type for complex as they really only are composition of entities combined with associated examples and they don't get extracted ... Or is it that slots are becoming complex ?

Reply:
Discussed offline

Will add a TODO with a notion of `Utterance Placeholder` (synonyms and variables)